storage/csv.go

```diff
@@ -196,6 +194,24 @@ func (c *CSVBatch) PersistModel(ctx context.Context, m interface{}) error {
 				continue
 			}
 
+			var encodeAsJSON bool
+
+			// Strings marked as json type are assumed to already be encoded
+			if fk != reflect.String && (t.types[i] == "json" || t.types[i] == "jsonb") {
+				encodeAsJSON = true
```

Comment:
Should this be false if its already encoded?

Reply:
No, this is checking if it has a json datatype but is not a string

Reply:
Comment should be clearer